common/landed_titles/wc_religious_titles.txt

```diff
@@ -72,6 +72,26 @@ d_church_scarlet_light = {
 	male_names = { Alexander Anastasius Benedictus Caelestinus Callistus Clemens Eugenius Leo Gregorius Hadrianus Honorius Innocentius Ioannes Lucius Marinus Martinus Nicolaus Sergius Silvester Stephanus Urbanus Victor Alonsus }
 }
 
+# kirin tor
+d_kirin_tor = {
```

Comment:
Check `Warcraft-Guardians-of-Azeroth-2\common\coat_of_arms\coat_of_arms\00_wc_landed_titles.txt`.
You may set CoA there. It may be like Dalaran but with different colors. Colors are in `Warcraft-Guardians-of-Azeroth-2\common\named_colors`.